front/components/PlansTables.tsx

```diff
@@ -180,6 +180,18 @@ export function ProPriceTable({
       size={size}
       magnified={false}
     >
+      {onClick && (!plan || plan.code !== PRO_PLAN_SEAT_29_CODE) && (
```

Comment:
This will also change the tables displayed to logged in admins, expected?

Reply:
@flvndvd 

So I don't know what this does.
I have nit written this, I have simply duplicated the action button already existing at the bottom of the sheet to have it also at the top of the list.

I trust that it was done properly before.